added a release blog post for 2.2.0 (#120)
KaotoIO/kaoto.io: content/blog/kaoto-2.2-release/index.md

---
title: Kaoto 2.2 released
date: 2024-09-20T00:00:00+06:00
summary: We are thrilled to announce the 2.2 release of Kaoto, the visual editor for Apache Camel! Check it out!
authors:
  - admin
tags:
  - Kaoto
  - Release
  - Kaoto 2.2
---
## Say "Hello" to Kaoto 2.2.0

> For this release we focused on enhancing the usability and also the look and feel of the graphical editor. While doing so we also fixed several reported issues, for instance the visualization of parallel steps inside a WireTap or LoadBalance EIP. We did also add a lot of icons for new and existing [Apache Camel](http://camel.apache.org) components, which hopefully help to better identify them in the catalog as well as in your integrations. This release also ships with [Apache Camel 4.8.0](https://camel.apache.org/blog/2024/09/RELEASE-4.8.0/) support.

Reviewer: Shouldn't we refer to multicast instead?

Reviewer: ```suggestion
For this release, we focused on enhancing the usability and the look and feel of the graphical editor. While doing so we also fixed several reported issues, for instance, the visualization of parallel steps inside the `Multicast` and `LoadBalance` EIPs. We also added new icons for existing and new [Apache Camel](http://camel.apache.org) components, which helps to better identify them in the catalog as well as in your integrations. This release also ships with [Apache Camel 4.8.0](https://camel.apache.org/blog/2024/09/RELEASE-4.8.0/) support.
```

Author: good catch
